Makefile.am

```diff
@@ -39,6 +40,7 @@ rdmainclude_HEADERS =
 
 # internal utility functions shared by in-tree providers:
 common_srcs =				\
+	src/hmem.c			\
```

Comment:
What are your thoughts on keeping device-specific ops separate (ie keep all the cuda ops in one file)? Just thinking that if we add more ops (open/close/get IPC handle, alloc/free, etc) and add more devices, this file might start to get a bit confusing.

Reply:
I could do something like:
- CUDA: src/cuda_hmem.c with ofi_cuda_hmem.h
- ROCr (AMD future work): src/rocr_hmem.c with ofi_rocr_hmem.h

Mainly, as long as I don't lose ofi_copy_from_hmem_iov() and ofi_copy_to_hmem_iov(), I am ok with any change.

Reply:
I would prefer separation if that's ok with you and @shefty 
I have been told I'm bad at naming things so I will let you or him figure out the best names for the files :P